database/models/User.js

```diff
@@ -0,0 +1,11 @@
+const bookshelf = require('./bookshelf');
+
+class User extends bookshelf.Model{
+  get tableName() {return 'users';}
+  get hasTimestamps() {return true;}
+  
+  contact() {
```

Comment:
This is still not plural. A User doesn't have a single contact. They can have many contacts.